src/main_menu.cpp

```diff
@@ -256,6 +256,11 @@ bool game::opening_screen()
     dirent *dp;
     DIR *dir;
 
+    if (!assure_dir_exist(FILENAMES["config_dir"].c_str())) {
```

Comment:
I wasn't sure about that, the existing code is pretty inconsistent and c_str is sometimes used, sometimes not. The check for savedir doesn't have it but the next one a couple of lines done for templatedir does have it.